src/navigation/wallet/screens/wallet-settings/WalletInformation.tsx

```diff
@@ -272,12 +346,21 @@ const WalletInformation = () => {
                     <SettingTitle>Copayer {index}</SettingTitle>
                   </InfoSettingsRow>
 
-                  <CopyButton onPress={() => copyText(xPubKey)}>
-                    <H7>{xPubKey}</H7>
-                  </CopyButton>
+                  <CopyRow
+                    onPress={() => {
+                      copyToClipboard(xPubKey);
+                      setCopiedXPubKey(xPubKey);
+                    }}>
+                    <H7 style={{width: '90%'}}>{xPubKey}</H7>
+                    <CopyImgContainerLeft style={{width: '10%'}}>
+                      {copiedXPubKey === xPubKey ? (
```

Comment:
For this one can you move the icon to right side of address? basically for left side screen text icon will be on end of the text and for right side screen text icon will be at the beginning.